cli/spec/kontena/client_spec.rb

```diff
@@ -233,6 +233,25 @@
       end
     end
 
+    context "with invalid response JSON" do
+      before :each do
+        allow(subject).to receive(:http_client).and_call_original
+
+        WebMock.stub_request(:any, 'http://localhost/v1/test').to_return(
+          status: 200,
+          headers: {
+            'Content-Type' => 'application/json',
+          },
+          body: 'this is not valid JSON',
+        )
+      end
+
+      it "raises Kontena::Errors::StandardError with 520" do
+        # XXX: for some reason the response.path is empty?
+        expect{subject.get('test')}.to raise_error(Kontena::Errors::StandardError, /Invalid response JSON from server for : JSON::ParserError/)
```

Comment:
Must be something related to webmock, the `response.path` does show up correctly for real requests?